<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendTokenPage</class>
 <widget class="QDialog" name="SendTokenPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="5" column="0">
      <widget class="QLabel" name="labelGasPrice">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>GasPrice</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelGasLimit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>GasLimit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="TokenAmountField" name="lineEditAmount"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditDescription"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPayTo">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>PayTo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QValidatedLineEdit" name="lineEditPayTo"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="lineEditGasLimit">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAmount">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelDescription">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="BitcoinAmountField" name="lineEditGasPrice"/>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelTokenBalanceText">
         <property name="text">
          <string>Balance:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelTokenBalance">
         <property name="text">
          <string>100.34</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="buttonsContainerWhite" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>qt/bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TokenAmountField</class>
   <extends>QLineEdit</extends>
   <header>qt/tokenamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditPayTo</tabstop>
  <tabstop>lineEditAmount</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>lineEditGasLimit</tabstop>
  <tabstop>lineEditGasPrice</tabstop>
  <tabstop>confirmButton</tabstop>
  <tabstop>clearButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
